hyperopt/fmin.py

```diff
@@ -234,7 +239,8 @@ def get_n_unfinished():
         ) as progress_ctx:
 
             all_trials_complete = False
-            while n_queued < N or (block_until_done and not all_trials_complete):
+            while (n_queued < N or (block_until_done and not all_trials_complete)) and \
+                    (self.timeout is None or (timer() - self.start_time) < self.timeout):
```

Comment:
this can be  `timer() - self.start_time < (self.timeout or float('inf'))`. Also, please use black to format code

Reply:
It can, but version I used (a) subjectively more readable, (b) objectively faster

Reply:
Black was applied